steps/plandek-deployment/1.0.0/step.yml

```diff
@@ -0,0 +1,61 @@
+title: Plandek - deployment
+summary: |
+  Enables you to track and visualize build and deployment events within Plandek's metrics and dashboards.
+description: |
+  Wrapper implementing the Plandek /deployment POST API. Used to complete a deployment within Plandek for a given release cycle.
+  More details can be found here: https://api.plandek.com/deployments/v1/ui/
+website: https://github.com/secretescapes/bitrise-step-plandek-deployment
```

Comment:
[recommendation] Please replace with https://plandek.com/.

Reply:
@lpusok what do you mean, which link should I replace with https://plandek.com/ ?

Reply:
Yes.